### PR TITLE
Add CI scripts and setup

### PR DESCRIPTION
**Pull request by jeffmaher**
Adds:

- GitHub Actions configurations for running CI scripts
- CI scripts placeholders (i.e. when we have things we want to run during PRs, we can add commands to the scripts)
- Version labeling GitHub Actions scripts, which will help us establish a version that can be referenced by downstream guides
- Removes a Text11ty funding file

This all goes towards #5.

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Run tests
+        run: docker run --rm ${{ github.repository }}:${{ github.head_ref }} sh ci/tests.sh
+      
+      # SECURITY
+      - name: Run security scans
+        run: docker run --rm ${{ github.repository }}:${{ github.head_ref }} sh ci/security.sh
+
+      # CODE QUALITY
+      - name: Run code quality scans
+        run: docker run --rm ${{ github.repository }}:${{ github.head_ref }} sh ci/code_quality.sh
```
Comment (SMakaiTakori) [inline .github/workflows/pull_request.yml:47]: I like that this is an option!

File: ci/semver.sh [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+SEMVER_REGEX="\
+^[vV]?\
+($NAT)\\.($NAT)\\.($NAT)\
+(\\-(${IDENT})(\\.(${IDENT}))*)?\
+(\\+${FIELD}(\\.${FIELD})*)?$"
+
+PROG=semver
+PROG_VERSION="3.1.0"
+
```
Comment (SMakaiTakori) [inline ci/semver.sh:23]: @jeffmaher I would love to talk more about this particular file when you have time and why REGEX is used, just curious!

**Review by SMakaiTakori — APPROVED** [review]
This looks great, @jeffmaher !

File: ci/semver.sh (hunk 2 above)
Comment (jeffmaher) [reply ci/semver.sh:23]: @SMakaiTakori : Happy to chat about it! With that said, this is entirely taken from someone else and just worked, with the original repo noted at the top of the file.

**Pull request merged**